src/test/java/bio/terra/service/dataset/DatasetAzureIntegrationTest.java

```diff
@@ -226,6 +258,37 @@ public void datasetsHappyPath() throws Exception {
     DatasetSummaryModel summaryModel2 =
         dataRepoFixtures.createDataset(
             steward, profileId, "it-dataset-omop.json", CloudPlatform.AZURE);
+
+    BillingProfileModel profileModel =
+        profileDao.getBillingProfileById(summaryModel2.getDefaultProfileId());
+    AzureStorageAccountResource storageAccountResource =
+        resourceService.getDatasetStorageAccount(
+            datasetService.retrieve(summaryModel2.getId()), profileModel);
+    AzureStorageAuthInfo storageAuthInfo =
+        AzureStorageAuthInfo.azureStorageAuthInfoBuilder(profileModel, storageAccountResource);
+
+    tableServiceClient =
+        azureAuthService.getTableServiceClient(
+            storageAuthInfo.getSubscriptionId(),
+            storageAuthInfo.getResourceGroupName(),
+            storageAuthInfo.getStorageAccountResourceName());
```

Comment:
You can simplify this as well:
```suggestion
        azureAuthService.getTableServiceClient(storageAuthInfo);
```